### DISCUSSION TITLE
Photo viewer

### DISCUSSION THREAD

**Discussion by ghost** [civitai/civitai#354] [Ideas]
Look for photos that people upload as reviews or just as models. Would be cool to browse through and see what tags people use and see other models that make the images look cool.

**Comment by ura23** [comment]
Very good suggestion. There is a lot good pictures generated, that posted by users, especially thus with promt. But it hard to find them, only accidentally so some kind of gallery will be very good.

**Comment by WASasquatch** [comment]
The model names, TI names, LORA names, etc, should all be being stored, so that way not only can one search civitai by a loras name in a prompt they found, but also, said images, could straight list out TIs/LORAs used that may have a presence on civitai.
